ext/opentelemetry-ext-jaeger/examples/jaeger_exporter_example.py

```diff
@@ -1,6 +1,6 @@
 import time
 
-from opentelemetry import trace
+from opentelemetry import trace, trace_api
```

Comment:
`trace_api` doesn't exist on opentelemetry, just `trace` will work fine.

Reply:
i see, in other places it gets imported as trace_api. Fixed.